internal/data/spicedb.go

```diff
@@ -269,6 +283,14 @@ func (s *SpiceDbRepository) Check(ctx context.Context, check *apiV0.CheckRequest
 	return &apiV0.CheckResponse{Allowed: apiV0.CheckResponse_ALLOWED_FALSE}, nil
 }
 
+func (s *SpiceDbRepository) IsBackendAvaliable() error {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	_, err := s.healthClient.Check(ctx, &grpc_health_v1.HealthCheckRequest{})
```

Comment:
Should check the response status here. A successful connection wouldn't get an error, but the response status could still be NOT_SERVING

Reply:
Added checks :)